lizmap/modules/lizmap/classes/lizmap.class.php

```diff
@@ -49,7 +52,30 @@ private function __construct()
      */
     public static function getServices()
     {
-        return jClasses::getService('lizmap~lizmapServices');
+        if (!isset(self::$lizmapServicesInstance)) {
+            $lizmapConfigTab = parse_ini_file(jApp::configPath('lizmapConfig.ini.php'), true);
+            $globalConfig = jApp::config();
+            $ldapEnabled = jApp::isModuleEnabled('ldapdao');
+            $varPath = jApp::varPath();
+            self::$lizmapServicesInstance = new lizmapServices($lizmapConfigTab, $globalConfig, $ldapEnabled, $varPath);
+        }
+
+        return self::$lizmapServicesInstance;
+    }
+
+    public static function saveServices()
+    {
+        $ini = new jIniFileModifier(jApp::configPath('lizmapConfig.ini.php'));
+        $liveIni = new jIniFileModifier(jApp::configPath('liveconfig.ini.php'));
+
+        $services = self::getServices();
+        $services->saveIntoIni($ini, $liveIni);
+
+        $modified = $ini->isModified() || $liveIni->isModified();
+        $ini->save();
+        $liveIni->save();
```

Comment:
```suggestion
        $modified = $ini->isModified() || $liveIni->isModified();
        if ($modified) {
            $ini->save();
            $liveIni->save();
        }
```
Is `save` necessary even if ini files are not modified ?

Reply:
no, not necessary

Reply:
My suggestion is not necessary, I have read that the old code save even if ini files are not modified